x-pack/plugins/security_solution/public/flyout/document_details/left/tabs/test_ids.ts

```diff
@@ -17,8 +17,10 @@ const INSIGHTS_TAB_TEST_ID = `${PREFIX}InsightsTab` as const;
 export const INSIGHTS_TAB_BUTTON_GROUP_TEST_ID = `${INSIGHTS_TAB_TEST_ID}ButtonGroup` as const;
 export const INSIGHTS_TAB_ENTITIES_BUTTON_TEST_ID =
   `${INSIGHTS_TAB_TEST_ID}EntitiesButton` as const;
+export const INSIGHTS_TAB_ENTITIES_TEST_ID = `${INSIGHTS_TAB_TEST_ID}Entities` as const;
 export const INSIGHTS_TAB_THREAT_INTELLIGENCE_BUTTON_TEST_ID =
   `${INSIGHTS_TAB_TEST_ID}ThreatIntelligenceButton` as const;
+export const INSIGHTS_TAB_PREVALENCE_TEST_ID = `${INSIGHTS_TAB_TEST_ID}Prevalence` as const;
```

Comment:
what do you think about renaming this to `INSIGHTS_TAB_PREVALENCE_LABEL_TEST_ID`?